### PR TITLE
feat: namespaced executor uri and jinaai prefix

### LINKED ISSUE #106: Parser for `jinahub+docker://name` doesn't work after introducing namespace
We need to make the parser compatiple for both `jinahub+docker://name/tag` and `jinahub+docker://username/name/tag`

### PR DESCRIPTION
**Pull request by delgermurun**
Resolves #106

### CODE REVIEW

File: hubble/executor/helper.py [hunk 1]
```diff
@@ -270,6 +270,39 @@ def parse_hub_uri(uri_path: str) -> Tuple[str, str, str, str]:
...
+        raise ValueError(f'{uri_path} is not a valid Hub URI.')
+
+    items = parser.path.strip('/').split(':')
+
+    if len(items) > 2:
+        raise ValueError(f'{uri_path} is not a valid Hub URI.')
+
+    name = items[0].strip('/')
+    tag = items[1].strip('/') if len(items) > 1 else None
+
```
Comment (samsja) [inline hubble/executor/helper.py:297]: can we make a private function out of this few lines ? And add unit test so see if it cover all of the corner case

**Review by samsja — COMMENTED** [review]
I adde a small sugestion

File: hubble/executor/helper.py (hunk 1 above)
Comment (girishc13) [reply hubble/executor/helper.py:297]: Using a class is even better to encapsulate the different operations on the URI.

Comment (delgermurun) [reply hubble/executor/helper.py:297]: IMO, it isn't worth the effort. More or less all edge cases have been tested.

**Review by nomagick — APPROVED** [review]

**Review by mapleeit — APPROVED** [review]
